build.sh

```diff
@@ -160,11 +160,16 @@ build_combined() {
     # Combine ar archives
     LIPO_OUTPUT="$out_path/$product_name/$module_name"
     xcrun lipo -create "$simulator_path/$binary_path" "$os_path/$binary_path" -output "$LIPO_OUTPUT"
-    if [[ "$destination" != "" ]] && [[ "$config" == "Release" ]]; then
+
+    if [[ $(xcode_major_version) != "6" && "$destination" != "" && "$config" == "Release" ]]; then
```

Comment:
I guess since we use `REALM_SWIFT_VERSION` to select the bitcode or non-bitcode version of core, it's sensible to use it here too.